components/brave_rewards/resources/android_page/actions/rewards_actions.ts

```diff
@@ -35,10 +37,24 @@ export const claimPromotion = (promotionId: string) => action(types.CLAIM_PROMOT
   promotionId
 })
 
+export const onClaimPromotion = (properties: Rewards.Captcha) => action(types.ON_CLAIM_PROMOTION, {
+  properties
+})
+
+export const attestPromotion = (promotionId: string, x: number, y: number) => action(types.ATTEST_PROMOTION, {
```

Comment:
this is not used on Android as Android doesn't have captcha

Reply:
Removed.